packages/atlas/src/components/NftCard/NftCard.tsx

```diff
@@ -27,7 +27,7 @@ export const NftCard: React.FC<NftCardProps> = ({
 }) => {
   return (
     <Container fullWidth={fullWidth}>
-      <VideoThumbnail clickable={false} thumbnailUrl={thumbnail.thumbnailUrl} />
+      <VideoThumbnail clickable={false} thumbnailUrl={thumbnail.thumbnailUrl} {...thumbnail} />
```

Comment:
I think we don't need `thumbnailUrl={thumbnail.thumbnailUrl}`  since there is spreading of `thumbnail`